Sources/SpeziOnboarding/OnboardingView.swift

```diff
@@ -75,28 +75,35 @@ public struct OnboardingView<TitleView: View, ContentView: View, ActionView: Vie
     private let contentView: ContentView
     private let actionView: ActionView
     
+    @Environment(\.isInOnboardingStack) private var isInOnboardingStack
```

Comment:
Mhh the SwiftUI `NavigationStack` doesn't enable something similar, right? I'm not super sure if we should deviate from that behavior.